### PR TITLE
[node-manager] Fix instance documentation

### PR DESCRIPTION
**Pull request by name212**
## Description

Fix instance documentation and add it to FAQ

## Why do we need it, and what problem does it solve?

Users does not understand what is Instance Resource

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager
type: chore
summary: Fix instance documentation.
impact_level: low
```

### CODE REVIEW

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -967,4 +967,17 @@ metadata:
     werf.io/track-termination-mode: NonBlocking
 ```
 
+## Что такое ресурс Instance?
+
+Ресурс Instance представляет собой ресурс эфемерной машины, независимый от реализации, например, машины созданные MachineControllerManager или Cluster API Provider Static будут иметь соотвествующий ресурс Instance.
```
Comment (RomanenkoDenys) [inline modules/040-node-manager/docs/FAQ_RU.md:972]:
```suggestion
Ресурс Instance представляет собой ресурс эфемерной машины, независимый от реализации, например, машины созданные MachineControllerManager или Cluster API Provider Static будут иметь соответcтвующий ресурс Instance.
```

File: modules/040-node-manager/crds/doc-ru-instance.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ spec:
       schema:
         openAPIV3Schema:
           description: |
-            Запрос на выделение облачного узла.
+            Независимый от реализации объект эфемерной машины.
```
Comment (z9r5) [inline modules/040-node-manager/crds/doc-ru-instance.yaml:25]:
```suggestion
            Описывает независимый от реализации объект эфемерной машины.
```

File: modules/040-node-manager/crds/instance.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ spec:
         openAPIV3Schema:
           type: object
           description: |
-            Defines the cloud instance.
+            An implementation-independent ephemeral machine resource.
```
Comment (z9r5) [inline modules/040-node-manager/crds/instance.yaml:26]:
```suggestion
            Describes an implementation-independent ephemeral machine resource.
```

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 4]
```diff
@@ -967,4 +967,17 @@ metadata:
     werf.io/track-termination-mode: NonBlocking
 ```
 
+## Что такое ресурс Instance?
+
+Ресурс Instance представляет собой ресурс эфемерной машины, независимый от реализации, например, машины созданные MachineControllerManager или Cluster API Provider Static будут иметь соответcтвующий ресурс Instance.
```
Comment (z9r5) [inline modules/040-node-manager/docs/FAQ_RU.md:972]:
```suggestion
Ресурс Instance содержит описание объекта эфемерной машины, без учета ее конкретной реализации. Например, машины созданные MachineControllerManager или Cluster API Provider Static будут иметь соответcтвующий ресурс Instance.
```

File: modules/040-node-manager/docs/FAQ.md [hunk 5]
```diff
@@ -964,4 +964,17 @@ metadata:
     werf.io/track-termination-mode: NonBlocking
 ```
 
+## What is an Instance resource?
+
+An Instance resource is an implementation-independent ephemeral machine resource; for example, machines created by MachineControllerManager or Cluster API Provider Static will have a corresponding Instance resource.
```
Comment (z9r5) [inline modules/040-node-manager/docs/FAQ.md:969]:
```suggestion
An Instance resource contains a description of an implementation-independent ephemeral machine resource. For example, machines created by MachineControllerManager or Cluster API Provider Static will have a corresponding Instance resource.
```

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 6]
```diff
@@ -967,4 +967,17 @@ metadata:
...
+
+Объект не содержит спецификации. Статус содержит ссылку на:
+1. InstanceClass если он существует для данной реализации.
+1. На объект Node kubernetes.
+1. Текущий статус машины.
+1. На этапе создания машины появляется информация как посмотреть [логи создания машины](#как-посмотреть-что-в-данный-момент-выполняется-на-узле-при-его-создании).
+
+При создании/удалении машины соотвственно создается/удаляется объект Instance.
+Самостоятельно ресурс Instance создать нельзя, но можно удалить. В таком случае машина будет удалена из кластера (процесс удаления зависит от деталей реализации.
```
Comment (z9r5) [inline modules/040-node-manager/docs/FAQ_RU.md:981]:
```suggestion
Объект не содержит спецификации. Статус содержит ссылку на:
1. InstanceClass, если он существует для данной реализации.
1. Объект Node Kubernetes.
1. Текущий статус машины.

Также, в статусе на этапе создания машины появляется информация о том, как посмотреть [логи создания машины](#как-посмотреть-что-в-данный-момент-выполняется-на-узле-при-его-создании).

При создании/удалении машины создается/удаляется соответствующий объект Instance.
Самостоятельно ресурс Instance создать нельзя, но можно удалить. В таком случае машина будет удалена из кластера (процесс удаления зависит от деталей реализации).
```

File: modules/040-node-manager/docs/FAQ.md [hunk 7]
```diff
@@ -964,4 +964,17 @@ metadata:
...
+
+The object does not contain a specification. The status contains a link to:
+1. InstanceClass if it exists for this implementation.
+1. To the Node kubernetes object.
+1. Current machine status.
+1. At the machine creation stage, information appears on how to view [machine creation logs](#how-do-i-know-what-is-running-on-a-node-while-it-is-being-created).
```
Comment (z9r5) [inline modules/040-node-manager/docs/FAQ.md:975]:
```suggestion
The object does not contain a specification. The status contains:
1. A link to the InstanceClass if it exists for this implementation;
1. A link to the Kubernetes Node object;
1. Current machine status;
1. Information on how to view [machine creation logs](#how-do-i-know-what-is-running-on-a-node-while-it-is-being-created) (available at the machine creation stage).
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6533](https://github.com/deckhouse/deckhouse/pull/6533) to the branch [release-1.54](https://github.com/deckhouse/deckhouse/tree/release-1.54) successful!

**Pull request merged**